src/main/kotlin/day15/Repository/Todos.Repository.kt

```
@@ -0,0 +1,36 @@
+            return parseJson(jsonDataSource)
+        }
+        set(value) {}
+
+    override suspend fun getTodos(): List<Todo> {
+        return todo
+    }
+
+    override suspend fun getCompletedTodos(): List<Todo> {
+        try {
+            return todo.filter { todo -> todo.completed }
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)